sub-packages/bionemo-size-aware-batching/pyproject.toml

```diff
@@ -11,9 +11,7 @@ requires-python = ">=3.10"
 license = {file = "LICENSE"}
 dynamic = ["version"]
 dependencies = [
-    # bionemo sub-packages
```

Comment:
I liked these... It's a nice way to let people know how you intended to organize.